packages/components/src/theme/navbar.scss

```diff
@@ -250,6 +250,25 @@
   border-radius: 0;
 }
 
+.container-nav {
+    padding: 8px 0px 8px 10px;
+    min-width: 100px;
+    color: #9d9d9d;
```

Comment:
should this use any of the gray color variables in our theme/variables.scss file? or should a new var be add3ed for this and the border-left below?

Reply:
The \#9d9d9d is to match the color for our navbar items that comes from the main theme, and there is no variable for it, and this is the only other usage so I'm not sure we want or need a variable. I can create a another variable for the border, since there are multiple usages of that value, all for the navbar.